From 7d1b33ab95714949a42144228ca180d998039755 Mon Sep 17 00:00:00 2001
From: "Matthew W. Thompson" <mattwthompson@protonmail.com>
Date: Tue, 2 May 2023 13:45:45 -0500
Subject: [PATCH] Remove unnecessary import

---
 openff/interchange/__init__.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/openff/interchange/__init__.py b/openff/interchange/__init__.py
index 6f4fb0ba..375a77ca 100644
--- a/openff/interchange/__init__.py
+++ b/openff/interchange/__init__.py
@@ -3,7 +3,6 @@ import importlib
 from types import ModuleType
 
 from openff.interchange._version import get_versions
-from openff.interchange.components.interchange import Interchange
 
 # Handle versioneer
 versions = get_versions()
-- 
2.40.1

